ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml

```diff
@@ -41,14 +45,121 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: lint
+  commands: |
+    export XDG_CACHE_HOME=/tmp/.cache
+    make lint
+  container:
+    clone: true
+    from: sro-test-bin
+- as: helm-lint
+  commands: |
+    make helm-lint
+  container:
+    clone: true
+    from: sro-test-bin
+- as: kube-lint
+  commands: |
+    make kube-lint
+  container:
+    from: src
 - as: unit
-  commands: make test
+  commands: make unit-test
   container:
     from: src
 - as: verify
   commands: make verify
   container:
     from: src
+- as: e2e-aws-ocp
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.9"
+  steps:
+    test:
+    - as: deploy-nfd
+      cli: latest
+      commands: |
+        git clone https://github.com/openshift/cluster-nfd-operator
```

Comment:
Please get the updated way to of deploying SRO https://github.com/openshift/release/blob/0ace79d9364f0cb64157ac8266c9f9f9caa77e7f/ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml#L89-L92